jupyterhub/spawner.py

```diff
@@ -539,8 +539,11 @@ def start(self):
     def stop(self, now=False):
```

Comment:
In the long run, 'force' might be a better name for this, since it is used for gentleness rather than time directly. Doesn't really matter tho.